### DISCUSSION TITLE
Why num_sites not equal to num_mutations?

### DISCUSSION THREAD

**Discussion by silvewheat** [tskit-dev/tskit#2205] [Q&A] [answered]
Hello,

I noticed that in my dataset, the num_sites is not equal to num_mutations. Is this as expected? Why they can be different?
![image](https://user-images.githubusercontent.com/19344997/165065994-7fe2dec3-edc4-455e-835c-6dbd47302265.png)

Best,
Yudong

**Comment by benjeffery** [comment, marked as answer]
Hi @silvewheat!
Thanks for your question. Mutations are placed on edges in the tskit data model (https://tskit.dev/tskit/docs/stable/data-model.html) so there can be an unlimited amount of mutations at a site, changing the genotypic state for the subset of nodes underneath the mutation.
